src/lib/alloc.c

```diff
@@ -666,7 +677,7 @@ static void *alloc_heap_buffer(struct mm_heap *heap, int zone, uint32_t caps,
 	return ptr;
 }
 
-/* allocates continuous buffers - not for direct use, clients use rballoc() */
+/* allocates continuous buffers - not for direct use, clients use rmalloc() */
```

Comment:
rballoc should be static.

Reply:
yes, I change rballoc to a static function